docs/contribute.md

```diff
@@ -1,328 +1,141 @@
 
-```bash
-$ make run-[BACKEND]
-```
+### 1. Creating a Discussion
 
-Substitute `[BACKEND]` by the backend name, _e.g._ `es` for Elasticsearch or
-`swift` for OpenStack Swift:
+- Use a clear and concise title that summarizes the topic.
+- In the description, provide context and details regarding the discussion.
+- Use labels to categorize the discussion (e.g., "question," "general discussion," "announcements," etc.).
 
-```bash
-# Start Elasticsearch local backend
-$ make run-es
-# Start Swift local backend
-$ make run-swift
-# Start all local backends
-$ make run-all
-```
+### 2. Participating in Discussions
 
-Now that you have started at least the `elasticsearch` and `swift` backends,
-it's time to play with them:
+- Engage in conversations respectfully, respecting others' opinions.
+- Avoid spamming or making off-topic comments.
+- Help answer questions when you can.
 
-```bash
-# Store a JSON file in the Swift backend
-$ echo '{"id": 1, "foo": "bar"}' | \
-    ./bin/ralph write -b swift -f foo.json
+### 3. Announcements
 
-# Check that we have created a new JSON file in the Swift backend
-$ bin/ralph list -b swift
-foo.json
+Project maintainers may use discussions to make announcements or share important updates.
+Be sure to follow these discussions to stay informed about the project's progress.
 
-# Read the content of the JSON file and index it in Elasticsearch
-$ bin/ralph read -b swift foo.json | \
-    bin/ralph write -b es
+### 4. Code of conduct 
 
-# Check that we have properly indexed the JSON file in Elasticsearch
-$ bin/ralph read -b es
-{"id": 1, "foo": "bar"}
-```
+Remember to maintain a positive and friendly atmosphere in discussions. 
+Use discussions as a space for productive interactions and community building.
 
-## Ralph's tray
+## Pull Requests (PR)
 
-Ralph is distributed along with its tray (a deployable package for Kubernetes
-clusters using [Arnold](https://github.com/openfun/arnold)). If you intend to
-work on this tray, please refer to Arnold's documentation first.
+Contributing to [Your Open Source Project Name] through pull requests is a powerful way to enhance the project. If you want to make changes or add new features, please follow these steps to submit a PR:
 
-Dependencies:
+### 1. Fork the Repository
 
-- [Kubectl](https://kubernetes.io/docs/tasks/tools/) (>`v.1.23.5`):
-  This CLI is used to communicate with the running Kubernetes instance you
-  will use.
-- [k3d](https://k3d.io/) (>`v.5.0.0`): This tool is used to set up
-  and run a lightweight Kubernetes cluster, in order to have a local
-  environment (it is required to complete quickstart instructions below to
-  avoid depending on an existing Kubernetes cluster).
-- [curl](https://curl.se/) is required by Arnold's CLI.
-- [gnupg](https://gnupg.org/) to encrypt Ansible vaults passwords and
-  collaborate with your team.
+Begin by forking Ralph project's repository. 
+This creates a copy of the project in your GitHub account that you can modify.
```

Comment:
```suggestion
```
I would assume people working on Ralph are familiar with forking.

Reply:
You're right